### DISCUSSION TITLE
Any plans for an edit/add TOC feature?

### DISCUSSION THREAD

**Discussion by rasputozen** [sumatrapdfreader/sumatrapdf#2494] [Q&A]
I know next to nothing about the pdf format so I don't know how much work this would require. Is this even a feature sumatra would add if it was feasible?

**Comment by GitHubRulesOK** [comment]
The addition of a ToC page is an editors nightmare for many apps
 but the Outline (Bookmarks) is only a fraction less of a problem and there are many similar issues for editor apps too.

There was a pretty good attempt for a means to export and manipulate .bkm files and reassemble with .vbkm editor, however  it too was problematic for many similar related reasons.

**Comment by kjk** [comment]
No plans at the moment.
